.github/workflows/build_and_release_gem.yml

```diff
@@ -40,7 +40,6 @@ jobs:
           TRACE_BUILD_TOKEN: ${{ secrets.ENOPS_5087_SOLARWINDS_APM_LIBOBOE }}
         run: |
           bundle exec rake fetch_oboe_file["prod"]
-          bundle exec rake oboe_verify
```

Comment:
What is the reason for removing this?

Reply:
Comparing the agent-binaries and github content always give me the failed case which was caused by new line detected on either from agent-binaries or github content.

Reply:
I see. Seems easy enough to try to compensate for the new line but looking more into this oboe_verify step, I do agree that it's unnecessary--how about let's remove it from the Rakefile too. And we'd no longer need the `TRACE_BUILD_TOKEN: ${{ secrets.ENOPS_5087_SOLARWINDS_APM_LIBOBOE }}` credential either.

Reply:
Updated